### PR TITLE
normalize span for 1-Click ML

### PR DESCRIPTION
**Pull request by sbettevy**
Closes #

Normalize span value for 1-Click ML
Span default value is 10
Apply formula max(10, 2h/time) where time is the group by value selected in the dataexplorer

  - [x] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

**Comment by sbettevy** [comment]
@regel formula has no upper limit (when time is close to 0 the span is infinite)

File: ui/src/loudml/utils/model.js [hunk 1]
```diff
@@ -1,22 +1,53 @@
...
+        Math.min(
+            duration,
+            60
+        )
     )
     return `${normalized}s`
 }
 
+export const normalizeSpan = bucketInterval => {
+    // span = min(10, (2h/bucketInterval))
```
Comment (regel) [inline ui/src/loudml/utils/model.js]: Sam, based on a few tests, we can select a value between 0 and 100:
span = max(
    10,
    min(24h/bucket_interval, 100)
)

**Review by regel — CHANGES_REQUESTED** [review]
Hi Sam, can you please update the span autoselect function?

File: ui/src/loudml/utils/model.js (hunk 1 above)
Comment (sbettevy) [reply ui/src/loudml/utils/model.js]: in next commit

**Review by regel — APPROVED** [review]

**Pull request merged**